manifests/nexus.yaml

```diff
@@ -33,5 +33,5 @@ spec:
   charts:
   - name: cray-nexus
     source: csm-algol60
-    version: 0.12.2
+    version: 0.12.*
```

Comment:
Temporary change to test glob processing. Represents a sample change which we may have in integration branch. Will be reverted before merge into release branch.